4788/src/main/include/Shooter.h

```diff
@@ -3,50 +3,35 @@
 #include "controllers/Controllers.h"
 #include "RobotMap.h"
 
-/**
- * State machine for the shooter in teleop
- * kAuto -> fot auto firing 
- * kManual -> driver controlled firing with a trigger
- * kStill -> Shooter is stationary. 
- * kTesting -> for testing :)
- */
 enum class TeleopShooter {
```

Comment:
Rename to `ShooterState`, it's more descriptive (the shooter is in states even outside of Teleop)